web/modules/custom/dbcdk_community/src/Form/ProfileEditForm.php

```diff
@@ -222,23 +297,61 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
         // to set the profiles ID to the body of the PUT request.
         $new_data['id'] = $this->profile->getId();
         $this->profile = $this->profileApi->profileUpsert(json_encode($new_data));
```

Comment:
This may cause an `ApiException` to be thrown. I do not think there is much use catching it here. In that case we should add a `@throws ApiException` to the DocBlock.